pom.xml

```diff
@@ -52,9 +52,10 @@
     <module>integration-tests</module>
   </modules>
   <scm>
-    <url>http://github.com/${project.scm.organisation}/${project.scm.repository}</url>
-    <connection>scm:git:git://github.com/${project.scm.organisation}/${project.scm.repository}.git</connection>
-    <developerConnection>scm:git:ssh://git@github.com/${project.scm.organisation}/${project.scm.repository}.git</developerConnection>
+    <connection>scm:git:https://github.com/Alfresco/alfresco-java-sdk.git</connection>
+    <developerConnection>scm:git:https://github.com/Alfresco/alfresco-java-sdk.git
+    </developerConnection>
```

Comment:
We can make closing tag at the same line

Reply:
done the changes.